lib/exsoda/configuration_reader.ex

```diff
@@ -49,11 +49,12 @@ defmodule Exsoda.ConfigurationReader do
   end
 
   defp get(path, r, json_opts) do
-    with {:ok, base} <- Http.base_url(r) do
+    with {:ok, base} <- Http.base_url(r),
```

Comment:
This is pretty cool. I recently read this thing about applicative functions. I think that's what this is? These two processes kick off simultaneously right?